core/backend/package.json

```diff
@@ -62,6 +62,7 @@
     "@itwin/core-geometry": "workspace:*",
     "@itwin/core-webpack-tools": "workspace:*",
     "@itwin/eslint-plugin": "^4.0.0-dev.33",
+    "@itwin/object-storage-azure": "^1.5.0",
```

Comment:
Why has this moved to `devDependencies`? It is used by `IModelHost`.

Reply:
ah this is a good catch, my bad. thought this was only being used for testing. will revert

Reply:
guess im getting old and my eyesight is waning
<img width="339" alt="image" src="https://user-images.githubusercontent.com/11051042/232153371-8b8039ed-a070-4273-82fd-1b38f52d208a.png">

updated